docs/source/user/notebook.md

```diff
@@ -1,21 +1,28 @@
 
 # Notebooks
 
-Jupyter Notebooks are documents that combine live runnable code, with narrative
-text (markdown), equations (LaTeX), images, interactive visualizations and other
+Jupyter Notebooks are documents that combine live runnable code with narrative
+text (Markdown), equations (LaTeX), images, interactive visualizations and other
 rich output:
 
 [screenshot]
 
 **Jupyter Notebook (.ipynb files) are fully supported in JupyterLab.**
 Furthermore, the [notebook document
 format](http://nbformat.readthedocs.io/en/latest/) used in JupyterLab is the
-same as the classic notebook. Thus, your existing notebooks should work fine in
+same as in the classic notebook. Your existing notebooks should work fine in
```

Comment:
"fine" --> "correctly"?

Reply:
or we could even just take out fine? or maybe use "well"?

Reply:
actually, of course, they may *not* work fine if they use notebook-specific things like javascript execution or rely on the notebook DOM, and I don't think we want to give the impression that a notebook will magically be "correct" :). How about "open correctly" - that's I think the standard we support :).